subsys/bluetooth/host/keys.c

```diff
@@ -493,13 +493,13 @@ void bt_keys_show_sniffer_info(struct bt_keys *keys, void *data)
 		sys_memcpy_swap(ltk, keys->ltk.val, keys->enc_size);
 		BT_INFO("SC LTK: 0x%s", bt_hex(ltk, keys->enc_size));
 	}
-
+#if !defined(CONFIG_BT_SMP_SC_PAIR_ONLY)
 	if (keys->keys & BT_KEYS_PERIPH_LTK) {
 		sys_memcpy_swap(ltk, keys->periph_ltk.val, keys->enc_size);
 		BT_INFO("Legacy LTK: 0x%s (peripheral)",
 			bt_hex(ltk, keys->enc_size));
 	}
-
+#endif
```

Comment:
```suggestion
#endif /* !CONFIG_BT_SMP_SC_PAIR_ONLY */ 

```

Reply:
@Casper-Bonde-Bose Please add the comment to clearly show which condition the `#endif` ends as per my suggestion

Reply:
Sorry, Accepted the suggested change, but it failed the checks - and I missed the comment part of your suggested change :-)